### ISSUE TITLE
Comments: Form is missing label element

### ISSUE THREAD

**Issue by SiobhyB** [Automattic/jetpack#5241]
Raised by @davidakennedy:

> The comment form area is missing a label element. The hidden textarea that expands the form appears to try and compensate for this, but it’s still not as intuitive for a screen reader user.

More information on the use of the label element can be [found here](https://developer.mozilla.org/en/docs/Web/HTML/Element/label).

**Comment by davidakennedy** [comment]
I did some more testing on this, prompted by a customer asking about it.

![screen shot 2017-08-21 at 4 50 14 pm](https://user-images.githubusercontent.com/1473618/29537774-fddec9c2-8690-11e7-9881-ca8e8502318c.png)

The label is actually in the source code, but not when you inspect it via dev tools (verified in Chrome and Firefox). That said, issues are still present because it looks like Autosize.js adds another `textarea` (see screenshot) so a screen reader announces it twice.

**Comment by charliescheer** [comment]
I had a user report this to me the other day in 15982611-hc

**Comment by AtrumGeost** [comment]
I was running some accessibility tests and also ran into this issue. I also noticed the social login icons are also missing a label and only using the title attribute:

![Screen Shot 2020-07-02 at 9 20 58 PM](https://user-images.githubusercontent.com/3696121/86425238-36519580-bcaa-11ea-985f-a2acdc976d61.png)

Please, let me know if this would need to be reported on a new issue or if is it fine to leave it here.
